airflow/providers/google/suite/transfers/gcs_to_gdrive.py

```diff
@@ -63,12 +63,12 @@ class GCSToGoogleDriveOperator(BaseOperator):
         For example, with prefix ``foo/*`` and destination_object ``blah/``, the file ``foo/baz`` will be
         copied to ``blah/baz``; to retain the prefix write the destination_object as e.g. ``blah/foo``, in
         which case the copied file will be named ``blah/foo/baz``.
+    :param destination_folder_id: The folder ID where the objects will be placed.
```

Comment:
It would be nice to see a small example of how this works in conjunction with destination `destination_object`.

Reply:
Thanks for the quick reply, does this example help?

Reply:
Yup, looks good now. :)